client/signup/README.md

```diff
@@ -191,4 +191,4 @@ handleSubmit = ( event ) => {
 }
 ```
 
-9 - open https://calypso.localhost:3000/start/hello in an incognito window. You should see your updated React component, and when you click the "Get started" button you should be taken to the next step.
+9 - open https://calypso.localhost:3000/start/hello in an incognito window. On opening you should be redirected to the first step showing your updated React component, and when you click the "Get started" button you should be taken to the next step, the user account creation form.
```

Comment:
I'd probably be slightly less specific & not make the change at the end (+`, the user account creation form`) just to minimize the ways this doc can get out of date again :)

Reply:
Thanks Jeff, good point, done.